apps/condo/domains/notification/UnsubscribeMiddleware.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+
+const { UNSUBSCRIBE_LINK_PATH } = require('@condo/domains/notification/constants/links')
```

Comment:
This const used only here. It's better to move it inside this file.

Reply:
I'll more constants there